photon-client/src/test/scala/com/linkedin/photon/ml/io/scopt/game/ScoptGameTrainingParametersParserTest.scala

```diff
@@ -164,7 +164,6 @@ class ScoptGameTrainingParametersParserTest {
       .put(GameTrainingDriver.treeAggregateDepth, treeAggregateDepth)
       .put(GameTrainingDriver.hyperParameterTuning, hyperparameterTuningMode)
       .put(GameTrainingDriver.hyperParameterTuningIter, hyperparameterTuningIter)
-      .put(GameTrainingDriver.hyperParameterTuningRange, hyperparameterTuningRange)
       .put(GameTrainingDriver.computeVariance, computeVariance)
```

Comment:
Should add custom ranges and alpha to this test

Reply:
Done.